src/frontend/e2e/tests/home.spec.ts

```diff
@@ -1,29 +1,29 @@
 import { expect, test } from "@playwright/test";
 import { getByTestID } from "../utils/selectors";
 
-//todo: populate the remaining elements
 const footer: Record<string, string> = {
   Github: "https://github.com/chanzuckerberg/czgenepi/",
   Careers:
     "https://chanzuckerberg.com/careers/career-opportunities/?initiative=science",
   "Learning Center":
     "https://help.czgenepi.org/hc/en-us/categories/6217716150804-Genomic-Epidemiology-Learning-Center",
+  //"Privacy":  "/privacy",
+  //"Terms":  "/terms",
+  //"Contact Us":  "mailto:hello@czgenepi.org"
 };
 
 test.describe("Home page tests", () => {
-  test("Should verify home page", async ({ page }, workerInfo) => {
-    const { baseURL } = workerInfo.config.projects[0].use;
-    await page.goto(`${baseURL}` as string);
-    await expect(page.locator(getByTestID("navbar-landing"))).toBeVisible();
+  test("Should verify home page", async ({ page }) => {
+    await page.goto("https://staging.czgenepi.org");
```

Comment:
you should be pulling this from the env, not hardcoding

Reply:
fixed